tools/mk-ca-bundle.pl

```diff
@@ -34,35 +34,85 @@
 use Getopt::Std;
 use MIME::Base64;
 use strict;
-use vars qw($opt_h $opt_i $opt_l $opt_q $opt_t $opt_v $opt_w);
+use vars qw($opt_h $opt_i $opt_l $opt_p $opt_q $opt_s $opt_t $opt_v $opt_w);
+use List::Util;
+use Text::Wrap;
 
-my $url = 'http://mxr.mozilla.org/mozilla/source/security/nss/lib/ckfw/builtins/certdata.txt?raw=1';
```

Comment:
There was some concern in https://github.com/joyent/node/pull/13860 that this wasn't `https`. Does this supersede that entirely?